benchmarks/core/is-decreasing.bril

```diff
@@ -0,0 +1,47 @@
+# Input: an array of length 6
```

Comment:
Looks like this may be out of date? The input appears to be an integer, not an array.

Reply:
Oops good catch. Just fixed this!